content/en/hub/hardware/expansion-shelves/es12-bsg.md

```diff
@@ -0,0 +1,10 @@
+---
+title: "ES12 Basic Setup Guide"
+linktitle: "ES12 BSG"
+description: "Guide included with ES12 expansion shelves purchased from iXsystems."
+---
+
+<object data="/pdf/ES12-2020-06-25.pdf" type="application/pdf" width="95%" height="1000">
+  Your browser does not support pdfs, <a href="/pdf/ES12-2020-06-25.pdf">click here to
```

Comment:
Almost all browsers support viewing a pdf. Think it would be worth removing the "browser does not support pdfs" part and making the title the link to the pdf? If not, then it's fine by me, go ahead and merge.

Reply:
the text only displays when the PDF viewer doesn't load for some reason - build preview doesn't show this because I think the link gets messed up with the different dir structure on the preview (local builds worked as expected).